proof/crefine/ARM/IsolatedThreadAction.thy

```diff
@@ -633,6 +633,18 @@ lemma getHWASID_isolatable:
             | simp add: page_directory_at_partial_overwrite)+
   done
 
+lemma getTCB_threadGet:
+  "do
+     tcbobj \<leftarrow> getObject t;
+     x \<leftarrow> f (atcbVCPUPtr (tcbArch tcbobj));
+     g x
+   od = do
+     vcpu_ptr \<leftarrow> threadGet (atcbVCPUPtr o tcbArch) t;
```

Comment:
I'm a bit confused by `atcbVCPUPtr` appearing in `ARM`.

Reply:
This was a derp, those variables are blue. Thanks for catching this!